src/test/java/com/cloudant/tests/util/Utils.java

```diff
@@ -33,4 +44,58 @@ public static String getHostName(String account) {
         }
 
     }
+
+    public static void removeReplicatorTestDoc(CloudantClient account, Response replicatorResponse)
+            throws Exception {
+
+        //Find replicator document for removal
+        ReplicatorDocument replicatorDoc = waitForReplicatorToComplete(account, replicatorResponse);
+
+        Response removeResponse = account.replicator()
+                .replicatorDocId(replicatorDoc.getId())
+                .replicatorDocRev(replicatorDoc.getRevision())
+                .remove();
+
+        assertThat(removeResponse.getError(), is(nullValue()));
+    }
+
+    public static ReplicatorDocument waitForReplicatorToComplete(CloudantClient account, Response
```

Comment:
The account variable is required to access the replicator.  The only way to not have the account passed to this method is to move the method outside of the Utils package and into this class.